### DISCUSSION TITLE
HLSL module gone in 2.5.6

### DISCUSSION THREAD

**Discussion by RobertMtx** [DiligentGraphics/DiligentEngine#312] [Q&A] [answered]
I just upgraded the library (I believe from v2.5.4 to v2.5.6), and noticed some of the files I used to link to are no longer present. One of the files is (was) `..\DiligentCore\ThirdParty\glslang\hlsl\Debug\HLSLd.lib`, but I'm assuming there may be others. Have these modules been renamed? And is there a document where I can find these types of changes? I wasn't able to locate anything about this in the ReleaseHistory.md file.

Thanks, appreciate any assistance!

**Comment by TheMostDiligent** [comment, marked as answer]
Yes, this module was removed from glslang. I don't think there are any other similar changes in third-party modules.

**Comment by RobertMtx** [comment]
I eventually found the change log in that package. For some reason, I was ignoring the `ThirdParty` part of the path. I'm actually not even sure why I was linking to this library. I'm still in the process of moving everything to a new PC, so will find out eventually.

Thanks, I appreciate the information!
